dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.faulttolerance-4.1.feature

```diff
@@ -1,8 +1,9 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=io.openliberty.org.eclipse.microprofile.faulttolerance-4.1
 singleton=true
-# io.openliberty.mpCompatible-x.x comes from io.openliberty.microprofile.cdi.api features
--features=io.openliberty.microprofile.cdi.api-4.0; ibm.tolerates:="4.1"
+-features=\
+ io.openliberty.mpCompatible-7.0,\
+ io.openliberty.microprofile.cdi.api-4.0; ibm.tolerates:="4.1"
```

Comment:
```suggestion
 io.openliberty.jakarta.cdi-4.0; ibm.tolerates:="4.1"
```

Reply:
For now we can do this, but for the EE 11 zip file, this will cause problems.  So you could make a io.openliberty.microprofile.faulttolerance.4.1.cdi.api-4.0 and 4.1 and do that instead.  Similar with the faulttolerate 4.0 you can do the same with changing 4.1 4.0 and the feature feature to 3.0 and 4.0 like it was with the existing io.openliberty.microprofile.cdi.api features if you want to me be more specific, or you can leave it with the io.openliberty.microprofile.cdi.api names and change to have the 4.0 one not tolerate mpCompatible 7.0.